addons/sys_intercom/stringtable.xml

```diff
@@ -4,7 +4,7 @@
         <Key ID="STR_ACRE_sys_intercom_infantryPhone">
             <English>Infantry Phone</English>
             <Spanish>Teléfono de infantería</Spanish>
-            <German>Gegensprechanlage</German>
+            <German>Infanterietelefon</German>
```

Comment:
During design phase of the component, we decided after consultation of actual Mech. Inf. Troops that the wording at Bundeswehr is `Gegensprechanlage`

Reply:
fixed